app/_components/commodities/components/FormResponseHeading.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { Flex, HStack, Heading, Image, Text, VStack } from '@chakra-ui/react';
+import { ICommodityFormResponse } from '@/app/_types/commodity';
+
+interface IFormResponseHeadProps {
+  commodityResponse: ICommodityFormResponse[];
+  legendItems: {
+    text: string;
+    src: string;
+    alt: string;
+  }[];
+}
+
+const FormResponseHeading: React.FC<IFormResponseHeadProps> = ({
+  commodityResponse,
+  legendItems,
+}) => (
+  <>
+    <Heading as="h2" size="md">
+      Commodity: {commodityResponse[0]?.commodityDisplayName}
```

Comment:
It is like you say, to display the name of the commodity back to the user that they filtered against. And yes it is available in a grandparent so it can be drilled down a couple levels if that's what you prefer. I thought it easier to just grab it from existing props as it involves no extra prop drilling nor test adjustment.